### PR TITLE
aa contrast standards for FAA family info

### PR DESCRIPTION
**Pull request by ATBull81**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [ ] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/186499202

# A brief description of the changes

Current behavior: Page was missing radio button navigation and had a large issue with the status display contrast

New behavior: Cucumber tests added, button navigation and usage for keyboards introduced, status display improved

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: CONTRAST_LEVEL_AA_IS_ENABLED

- [ ] DC
- [x] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

Original on trunk:
trunk

Original on branch (to show changes haven't modified anything):
withoutchanges

New:
Screenshot 2023-12-18 at 12 33 14 PM

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/javascript/css/contrast_level_aa.scss [hunk 1]
```diff
@@ -36,3 +38,31 @@
...
+  margin-right: 5px !important;
+}
+.warning-icon {
+    background-image: url('../../assets/images/warning-icon.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    color: transparent !important;
+    font-size: 27px !important;
+  }
+.in-progress-icon {
```
Comment (ATBull81) [inline app/javascript/css/contrast_level_aa.scss:63]: Can add a reason why we do this here if needed. In short, we don't want to show the alternate icons when we are using this new theming so we turn them transparent.

Comment (ATBull81) [reply app/javascript/css/contrast_level_aa.scss:66]: Added note: https://github.com/ideacrew/enroll/pull/3398/commits/beac84742e39cea449a9c117105f55c6597be94c

I wasn't able to find a better way, but if you can think of one please let me know. Didn't want to add any conditional checks in [_status](https://github.com/ideacrew/enroll/pull/3398/files#diff-d9e6ef847a39de598b03fb848b804b6a64bfbd4ee03b77daf3d1e6a4355b2c3fR23) or [info_needed](https://github.com/ideacrew/enroll/pull/3398/files#diff-bbcd5a45b4e20e6e67ab5a86773762f688a612e4b8bcc1e0b4c6cb19ef894639R4)

Comment (ATBull81) [reply app/javascript/css/contrast_level_aa.scss]: fixed here: https://github.com/ideacrew/enroll/pull/3398/commits/dc15ac6b0625a4f8572df90df49e29349beb3f8d#diff-0f51e756c58b7b7a714c9b46ee136577e97e38a2307b1e57a12f80523995e0baL64-L71

**Comment by ATBull81** [comment]
https://github.com/ideacrew/enroll/pull/3398/commits/dc15ac6b0625a4f8572df90df49e29349beb3f8d
added fallback values for ::before icons
Screenshot 2023-12-19 at 2 19 42 PM

**Review by scaustin34 — DISMISSED** [review]
Looks good to me!

**Review by mdkaraman — APPROVED** [review]
Looks good!

**Review by scaustin34 — APPROVED** [review]

**Pull request merged**